libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/helpers/RecyclerViewScrollPositionSaver.java

```diff
@@ -0,0 +1,10 @@
+package org.wordpress.android.util.helpers;
```

Comment:
Unless we think we're going to add another class that implements this interface, I think we can simplify things by dropping it.